server/src/api/observation/observation.model.ts

```diff
@@ -41,14 +40,10 @@ export class Observation extends BaseEntity {
     { primary: false, onDelete: 'RESTRICT', eager: true, orphanedRowAction: 'nullify', cascade: false })
   type!: ReportSchema;
 
-  /* military unit description
-   Initially we will not make this a FK (foreign key) but it is to be considered.
-   If made a FK then we need to make sure we have all the FKs in the database and
-   also we would need to handle any resulting errors if a given FK is not found */
-  @Column({ length: 300 })
-  unitId!: string;
-
-  // military unit description
+  // military unit
   @Column({ length: 300 })
   unit!: string;
+
+  @Column({ length: 100})
+  reportingGroup?: string;
```

Comment:
TypeORM doesn't pick up on the `?` when defining the schema, so you may have to do this to make sure the column is nullable (assuming that's the intention)...

```typescript
  @Column({
    length: 100,
    nullable: true,
  })
  reportingGroup?: string;
```